django_extras/contrib/auth/models.py

```diff
@@ -129,7 +129,8 @@ class Document(SingleOwnerMixin, models.Model):
 
     """
     owner = models.ForeignKey(USER_MODEL_NAME, verbose_name=_('owner'),
-                              related_name='%(app_label)s_%(class)s_owner')
+                              related_name='%(app_label)s_%(class)s_owner',
+                              on_delete=models.CASCADE, null=True, blank=False)
```

Comment:
Is there a reason for null=True?

Reply:
It's the only way 'makemigrations' would work for me; but perhaps there is a better way?